src/ui-kit/Select/Select.tsx

```diff
@@ -39,26 +40,43 @@ export const Select = JSX<SelectProps>(
   }) => {
     const [isOpen, { setFalse: close, toggle }] = useBool();
     const [searchValue, setSearchValue] = useState('');
+    const popupSelect = useSelectPopup();
```

Comment:
можно этот интерфейс сюда перенести приватно:
```ts
interface SelectPopupOptions {
  top: number;
  left: number;
  width: number;
}
```

```suggestion
    const popup = usePopupManager<SelectPopupOptions>();
```